server/src/weaverbird/backends/pypika_translator/translators/postgresql.py

```diff
@@ -30,6 +30,7 @@ class PostgreSQLTranslator(SQLTranslator):
         integer="INTEGER",
         text="TEXT",
         datetime="TIMESTAMP",
+        timestamp="TIMESTAMP",
```

Comment:
Could you please set the `TO_DATE_OP` to `TO_TIMESTAMP` ?

Reply:
good catch.